### PR TITLE
Add units in the audit log header

### PR DESCRIPTION
**Pull request by nitisht**
This allows removing the `ns` from the field, allowing better plots of these fields in log management systems.

### CODE REVIEW

File: logger/message/audit/entry.go [hunk 1]
```diff
@@ -35,17 +35,19 @@ type Entry struct {
...
+		Bucket                  string          `json:"bucket,omitempty"`
+		Object                  string          `json:"object,omitempty"`
+		Objects                 []ObjectVersion `json:"objects,omitempty"`
+		Status                  string          `json:"status,omitempty"`
+		StatusCode              int             `json:"statusCode,omitempty"`
+		InputBytes              int64           `json:"rx"`
+		OutputBytes             int64           `json:"tx"`
+		HeaderBytes             int64           `json:"txHeaders,omitempty"`
+		TimeToFirstByte         string          `json:"timeToFirstByte,omitempty"`
+		TimeToFirstByteWithUnit string          `json:"timeToFirstByte(ns),omitempty"`
```
Comment (Praveenrajmani) [inline logger/message/audit/entry.go:48]:
```suggestion
		TimeToFirstByteInNS string          `json:"timeToFirstByteInNS,omitempty"`
```

File: logger/message/audit/entry.go [hunk 2]
```diff
@@ -35,17 +35,19 @@ type Entry struct {
...
+		TimeToResponse          string          `json:"timeToResponse,omitempty"`
+		TimeToResponseWithUnit  string          `json:"timeToResponse(ns),omitempty"`
```
Comment (Praveenrajmani) [inline logger/message/audit/entry.go:50]:
```suggestion
		TimeToResponseInNS  string          `json:"timeToResponseInNS,omitempty"`
```

**Review by Praveenrajmani — COMMENTED** [review]
can we indicate that the unit here is `ns` in the field names?

**Comment by nitisht** [comment]
> can we indicate that the unit here is `ns` in the field names?

Yes, this is actually better. Fixing now. Thanks @Praveenrajmani

File: logger/message/audit/entry.go [hunk 3]
```diff
@@ -35,17 +35,19 @@ type Entry struct {
...
+		Objects                 []ObjectVersion `json:"objects,omitempty"`
+		Status                  string          `json:"status,omitempty"`
+		StatusCode              int             `json:"statusCode,omitempty"`
+		InputBytes              int64           `json:"rx"`
+		OutputBytes             int64           `json:"tx"`
+		HeaderBytes             int64           `json:"txHeaders,omitempty"`
+		TimeToFirstByte         string          `json:"timeToFirstByte,omitempty"`
+		TimeToFirstByteInNS string          `json:"timeToFirstByteInNS,omitempty"`
+		TimeToResponse          string          `json:"timeToResponse,omitempty"`
+		TimeToResponseInNS  string          `json:"timeToResponseInNS,omitempty"`
```
Comment (Praveenrajmani) [inline logger/message/audit/entry.go:50]: we need to run `go fmt` on this file

**Review by Praveenrajmani — APPROVED** [review]
Just a minor nit. LGTM otherwise

File: logger/message/audit/entry.go (hunk 3 above)
Comment (nitisht) [reply logger/message/audit/entry.go]: done

**Comment by nitisht** [comment]
cc @harshavardhana could you please take a look..

**Pull request merged**